internal/commands/vaultsecrets/secrets/create.go

```diff
@@ -127,11 +135,70 @@ type secretConfigInternal struct {
 }
 
 type SecretConfig struct {
-	Type            integrations.IntegrationType `hcl:"type"`
-	IntegrationName string                       `hcl:"integration_name"`
-	Details         cty.Value                    `hcl:"details"`
+	Type    integrations.IntegrationType `hcl:"type"`
+	Details cty.Value                    `hcl:"details"`
 }
 
+var (
```

Comment:
Overall, this implementation feels pretty hacky, but allowing the interactive experience from hardcoded values for now is a bit tricky. There's a function [here](https://github.com/hashicorp/hcp/pull/167/files#diff-7f92b17a95650abeca3ec432e8bcf6859f19709d1bdb6b3687fb5d704940a4ecR542) that depends on the type of the empty values in the maps below. Definitely open to an alternative solution that's either less code or not as hacky.